docker-io.spec

```diff
@@ -219,12 +224,12 @@ Conflicts: selinux-policy
 BuildRequires: selinux-policy
 BuildRequires: selinux-policy-devel
 Requires: policycoreutils
```

Comment:
I'm guessing we can remove this too, as we already have this for `Requires(post)`